packages/ui-core/shared/src/lib/table-components/people-list/people-list.component.html

```
@@ -0,0 +1,44 @@
+<!--
+  One horizontal group per cell: named people first, then avatar-only people,
+  then a `+N` chip. No per-person box and no per-person border — see
+  `people-list.component.scss`.
+-->
+@if (total > 0) {
+	<div class="people" [class.is-wrapped]="wrap">
+		@for (person of named; track person.key) {
+			<a class="person" [class.is-link]="!!person.id" [nbTooltip]="person.name" (click)="onSelect(person)">
+				<span class="person-avatar">
+					@if (person.imageUrl) {
+						<img [src]="person.imageUrl" [alt]="person.name" (error)="onImageError(person)" />
```

HTMLHint (1.9.2)

[warning] An alt attribute must be present on <img> elements.

(alt-require)